api/cli/cli.go

```diff
@@ -42,6 +42,9 @@ func (*Service) ParseFlags(version string) (*portainer.CLIFlags, error) {
 		TLSCacert:         kingpin.Flag("tlscacert", "Path to the CA").Default(defaultTLSCACertPath).String(),
 		TLSCert:           kingpin.Flag("tlscert", "Path to the TLS certificate file").Default(defaultTLSCertPath).String(),
 		TLSKey:            kingpin.Flag("tlskey", "Path to the TLS key").Default(defaultTLSKeyPath).String(),
+		SSL:               kingpin.Flag("ssl", "SSL support").Default(defaultSSL).Bool(),
+		SSLCert:           kingpin.Flag("sslcert", "SSL cert").Default(defaultSSLCertPath).String(),
```

Comment:
Change `SSL cert` to `Path to the SSL certificate used to secure the Portainer instance`